templates/boltzmann.js

```diff
@@ -2528,12 +2527,15 @@ async function redisReachability (context, meta) {
 // {% endif %}
 
 // - - - - - - - - - - - - - - - -
-// Decorators
+// Validators
 // - - - - - - - - - - - - - - - -
-function validateBody(schema) {
-  ajv = ajv || require('ajv')
-  ajvStrict = ajvStrict || new ajv()
-  const validator = ajvStrict.compile(schema && schema.isFluentSchema ? schema.valueOf() : schema)
+function validateBody(schema, {
+  ajv = addAJVFormats(addAJVKeywords(new AJV({
+    useDefaults: true,
+    strictTypes: isDev() ? true : "log",
```

Comment:
Editorializing here: AJV sets `strictTypes` to log by default. I disagree with this setting: if we're going to complain about it, I want people to fix it before taking it to prod. Thus, we throw exceptions on missing or incorrect type information in schemas in development mode and only log in prod.